crm_compassion/migrations/14.0.1.3.0/pre-migration.py

```diff
@@ -0,0 +1,13 @@
+from openupgradelib import openupgrade
+
+
+@openupgrade.migrate()
+def migrate(env, version):
+    openupgrade.logged_query(
+        env.cr,
+        """
+        UPDATE crm_stage
+        SET is_lost = TRUE
+        WHERE LOWER(name) LIKE '%lost%';
+        """,
+    )
```

Comment:
This should also be in post-migrate file, as the first step. Because the column is_lost doesn not yet exist in pre-migration.

Reply:
Oh yeah, you are right. Moved to post-migration